lib/citriage.rb

```diff
@@ -74,39 +79,70 @@ def list_jobs json
       mod_status = true
       failed_job = String.new
       begin
-        json['jobs'].each do |job|
-          if job['color'] == 'red'
-            mod_status = false
-            failed_job = job['url']
-            break
-          else
-            mod_status = true
+        json.each do |name, branch|
```

Comment:
This loop is going through both branches with master first, so if release is green, mod_status ends up true and the short view will report the module as green. My thought is we should leave the quick view to just master for now, since that's the main focus anyway. What do you think?